### PR TITLE
added person class

### PR DESCRIPTION
**Pull request by SURTHANILOKESH**
with instance variables

### CODE REVIEW

File: src/main/java/uitl/DbUtil.java [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package uitl;
 
 public class DbUtil {
-
+	public static void main(String[] args) {
+		System.out.println("login branch");
```
Comment (vishnuramireddy) [inline src/main/java/uitl/DbUtil.java:5]: Is main method required

File: src/main/java/uitl/Person.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package uitl;
```
Comment (vishnuramireddy) [inline src/main/java/uitl/Person.java:1]: Why person class inside util??

File: src/main/java/uitl/DbUtil.java (hunk 1 above)
Comment (SURTHANILOKESH) [reply src/main/java/uitl/DbUtil.java:5]: yes it is required

File: src/main/java/uitl/Person.java (hunk 2 above)
Comment (SURTHANILOKESH) [reply src/main/java/uitl/Person.java:1]: tell me sir where i have to insert

**Pull request merged**